### DISCUSSION TITLE
Recovery account

### DISCUSSION THREAD

**Discussion by xelyos94ro** [community/community#22110] [New to GitHub]
<p>Hello, is there a way to recover my account? I was using Windows 10 and now i switch to Ubuntu. The problem is i forgot to save email and password and i don’t know how to recovery my account.</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>Hi there! :wave: Welcome to the Community!</p>
<p>Hi there,</p>
<p>If you’ve forgotten your password, you can reset it by typing your email address here:</p>
<aside class="onebox whitelistedgeneric">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">
      <a href="https://github.com/password_reset" target="_blank">GitHub</a>
  </header>
  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/11161530/181068910-2edd0959-451b-4079-b684-870f6c345fb3.png" class="thumbnail onebox-avatar" width="500" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_500x500.png, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_750x750.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_1000x1000.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_10x10.png">

<h3><a href="https://github.com/password_reset" target="_blank">Build software better, together</a></h3>

<p>GitHub is where people build software. More than 50 million people use GitHub to discover, fork, and contribute to over 100 million projects.</p>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>I hope that helps!</p>

**Comment by TauTech99** [comment]
Hello how to recover a Github organazation deleted with its repository ?
